### PR TITLE
Updates issues, and adds checks for comments too

### PR DESCRIPTION
**Pull request by orta**
Migrates to use the new danger file syntax ( export a default func ) and uses the "new" [Webhooks types](https://github.com/orta/github-webhook-event-types) - it will also edit issue comments too now.

### CODE REVIEW

File: org/issue.ts [hunk 1]
```diff
@@ -1,45 +1,30 @@
...
-
-// Support quietly transforming issue bodies from "XCode" to "Xcode".
-if (issue.body.includes("XCode")) {
-  const newBody = issue.body.replace(/XCode/g, "Xcode");
+  const newBody = issue.body.replace(/ Cocoapods /g, " CocoaPods ")
+                            .replace(/ XCode /g, " Xcode ");
 
-  schedule(async () => {
+  // Update the comment
+  if (newBody !== issue.body) {
```
Comment (amorde) [inline org/issue.ts:18]: nice!

**Review by amorde — COMMENTED** [review]
Looks like `org-wide` and `Orgs` need to be added to the spellcheck ignore list

File: org/comment.ts [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  const comment = webhook.comment
+  const repo = webhook.repository
+
+  // Support quietly transforming comment bodies from "Cocoapods" to "CocoaPods".
+  // Wrap it in whitespace to avoid changing links.
+  
+  // > "aCocoapods CocoaPods Cocoapods ".replace(/ Cocoapods /g, " CocoaPods ")
+  // => 'aCocoapods CocoaPods CocoaPods '
+
+  const newBody = comment.body.replace(/ Cocoapods /g, " CocoaPods ")
```
Comment (endocrimes) [inline org/comment.ts:14]: Super, super pedantic, but could we pull this into a map of query: replacement?

File: org/comment.ts [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+                              .replace(/ XCode /g, " Xcode ");
+
+  // Update the comment
+  if (newBody !== comment.body) {
+    await danger.github.api.issues.editComment({
```
Comment (endocrimes) [inline org/comment.ts:19]: nice 👍

File: org/issue.ts [hunk 4]
```diff
@@ -68,4 +53,7 @@ schedule(async () => {
     }
 
   } catch (error) {}
-});
+
+}
+
+
```
Comment (endocrimes) [inline org/issue.ts]: rm extra line?

File: org/comment.ts (hunk 2 above)
Comment (orta) [reply org/comment.ts:14]: Maybe once it hits three, abstracting at twice feels a bit too early

File: org/issue.ts (hunk 4 above)
Comment (orta) [reply org/issue.ts]: good spot, don't have prettier auto-formatting everything, argh

**Review by endocrimes — APPROVED** [review]

**Pull request merged**